From: ChepKun <ProgrammerKun@gmail.com>
Subject: [PATCH] av: fix omx and video recording

diff --git a/media/libstagefright/Android.mk b/media/libstagefright/Android.mk
index d02e1a7..9585d2f 100755
--- a/media/libstagefright/Android.mk
+++ b/media/libstagefright/Android.mk
@@ -89,7 +89,8 @@ LOCAL_C_INCLUDES += \
         $(TOP)/external/openssl/include \
         $(TOP)/hardware/qcom/media/mm-core/inc \
         $(TOP)/system/core/include \
-        $(TOP)/hardware/qcom/$(DISPLAY)/libgralloc
+        $(TOP)/hardware/qcom/$(DISPLAY)/libgralloc \
+	$(TOP)/device/huawei/$(TARGET_DEVICE)/qcom-hardware/display-legacy/libgralloc
 
 ifeq ($(BOARD_USES_STE_FMRADIO),true)
 LOCAL_SRC_FILES += \
diff --git a/media/libstagefright/colorconversion/Android.mk b/media/libstagefright/colorconversion/Android.mk
index af8ff57..420e706 100644
--- a/media/libstagefright/colorconversion/Android.mk
+++ b/media/libstagefright/colorconversion/Android.mk
@@ -15,7 +15,8 @@ endif
 LOCAL_C_INCLUDES := \
         $(TOP)/frameworks/native/include/media/openmax \
         $(TOP)/hardware/msm7k \
-        $(TOP)/hardware/qcom/$(DISPLAY)/libgralloc
+        $(TOP)/hardware/qcom/$(DISPLAY)/libgralloc \
+	$(TOP)/device/huawei/$(TARGET_DEVICE)/qcom-hardware/display-legacy/libgralloc
 
 LOCAL_MODULE:= libstagefright_color_conversion
 
